andela/ah-centauri-frontend
# chore(host-heroku): deploy frontend to heroku

[Delivers #165817647]
#### What does this PR do?
Hosting frontend on the heroku server.

#### Description of Tasks to be completed?
Hosts the front-end of the Author's Haven web application on the heroku staging environment.

#### How should this be manually tested?
- Clone the project: `git clone https://github.com/andela/ah-centauri-frontend`
- cd into the project folder
- checkout to branch `chore/165817647-host-heroku`
- Install dependencies: `npm install`
- Launch development server: `npm dev`
- Opens the homepage on the browser you should see the navbar, body and the footer.
- Launch production server: `npm start`
- Open heroku link by: [Clicking Here](https://ah-centauri-frontend-staging.herokuapp.com)
- navigate to the login and/or sign up pages on the top right navigation bar

#### Type of change
 - [x] chore(a non-breaking change that sets up react router).

#### What are the relevant Pivotal Tracker Stories?
[#165817647](https://www.pivotaltracker.com/story/show/165817647)

Assist @bl4ck4ndbr0wn Alpha Nganga
